install_scripts/setup_sadm.sh

```diff
@@ -12,6 +12,27 @@ Server = https://repo.prologin.org/
 EOF
 fi
 
+sed -i 's/^[#]IgnorePkg.*$/# Ignoring all of the following packages\
+# Install utilities\
+IgnorePkg = arch-install-scripts syslinux mkinitcpio-nfs-utils\
+# Boot and file systems utilities\
+IgnorePkg = nbd tftp-hpa conntrack-tools openresty ipxe-sadm-git nfs-utils\
+# Network\
+IgnorePkg = openssh dhcp bind ipset dnsutils rsync tcpdump\
+# Useful tools\
+IgnorePkg = vim rxvt-unicode-terminfo git strace wget ethtool tree mtr iperf\
+IgnorePkg = atop htop iotop iftop nethogs jq tmux bash-eternal-history\
+# Monitoring\
+IgnorePkg = grafana prometheus prometheus-node-exporter-git\
```

Comment:
Same, not sure it's a good idea to pin these (`prometheus-node-exporter-git` is a custom package, which we _may_ need to upgrade)

Reply:
Fixed according to what was said in #120